test/clj/cider/nrepl/middleware/info_test.clj

```diff
@@ -424,6 +424,17 @@
       (let [response (session/message {:op "eldoc" :symbol "." :ns "user"})]
         (is (= (:status response) #{"done"}))))
 
+    (testing "java interop method with multiple classes"
+      (let [response (session/message {:op "eldoc" :symbol ".length" :ns "cider.nrepl.middleware.info-test"})]
+        (is (= (:class response)
+               "java.lang.String, java.lang.StringBuffer, java.lang.CharSequence, java.lang.StringBuilder"))
```

Comment:
Yeah .. That's what I realised later. It would allow us to handle this `join` in emacs and also keep the separator configuration away from the middleware. I guess I wanted to keep the interface between emacs and middleware constant. Always a string. But it needs to be a list. Do you think we should also return an array when there is a single match ?